packages/twenty-server/src/core/user-workspace/user-workspace.auto-resolver-opts.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  AutoResolverOpts,
+  ReadResolverOpts,
+  PagingStrategies,
+} from '@ptc-org/nestjs-query-graphql';
+
+import { UserWorkspace } from 'src/core/user-workspace/user-workspace.entity';
+
+export const userWorkspaceAutoResolverOpts: AutoResolverOpts<
```

Comment:
@AdityaPimpalkar I don't think we should expose these endpoints on our API, they will expose a surface that could be used by security breaches